build/squid/local.mog

```diff
@@ -0,0 +1,40 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+# Copyright 2020 Carsten Grzemba
+
+license COPYING license=GPLv2
+
+<include binlink.mog>
+<include manlink.mog>
+
+group groupname=$(GROUP) gid=84
+user ftpuser=false username=$(USER) uid=84 group=$(GROUP) \
```

Comment:
gid 84 collides with `nagcmd`. Please have a look at `doc/idlist.md` and select a free uid/gid. Then please document it in `doc/idlist.md`